lib/urlwatch/reporters.py

```diff
@@ -161,37 +161,29 @@ def submit(self):
     def _parts(self):
         cfg = self.report.config['report']['html']
 
-        yield SafeHtml("""<!DOCTYPE html>
-        <html><head>
-            <title>urlwatch</title>
-            <meta http-equiv="content-type" content="text/html; charset=utf-8">
-            <meta name="viewport" content="width=device-width, initial-scale=1.0">
-            <style type="text/css">
-                body { font-family: sans-serif; }
-                .diff_add { color: green; background-color: lightgreen; }
-                .diff_sub { color: red; background-color: lightred; }
-                .diff_chg { color: orange; background-color: lightyellow; }
-                .unified_add { color: green; }
-                .unified_sub { color: red; }
-                .unified_nor { color: #333; }
-                table { font-family: monospace; }
-                h2 span.verb { color: #888; }
-            </style>
-        </head><body>
-        """)
+        yield SafeHtml(
+            '<!DOCTYPE html>\n'
+            '<html>\n'
+            '<head>\n'
+            '<title>urlwatch</title>\n'
+            '<meta http-equiv="content-type" content="text/html; charset=utf-8">\n'
+            '<meta name="viewport" content="width=device-width, initial-scale=1.0">\n'
+            # No stylesheet: styles are embedded to avoid overwriting by email clients
+            '</head>\n'
+            '<body>\n')
```

Comment:
Can you please keep the multi-line string here, even if you removed the style sheets?

Reply:
Whitespace in HTML doesn't matter. If it is a problem use [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) on the multi-line string.